docs/tutorial/managing-resource-usage.md

```diff
@@ -0,0 +1,29 @@
+# Managing resource usage
+
+The charm can be hardware resource instensive. As each instance of self-hosted runner application is hosted in a virtual machine.
+
+## Virtual machine resource usage
+
+The minimum resource usage for a single virtual machine:
```

Comment:
s/minimum resource usage for a single virtual machine/minimum requirements for a single virtual machine are